src/rss/compile-rss-feed.ts

```diff
@@ -39,9 +40,24 @@ function generateRssItem(geojsonFeature: Feature, feedTemplate: RssDatasetTempla
     feedTemplateTransforms
   );
 
+  // if georss element is present in rss feed template
+  // use georss from template else generate georss from
+  // geojson
```

Comment:
I don't fully understand this comment. As far as I can tell, the only this branch does is conditionally remove the `item.georss:where` field from the template. Is there another place in the code that the georss is generated? If so, we should point devs to that location.

Reply:
If the `if` block is not executed (i.e. if georss element is not present in the feed template), code [here](https://github.com/koopjs/koop-output-rss/blob/b/11748-rss-feeds/src/rss/compile-rss-feed.ts#L55-L57) after the block is executed which generates georss from geojson. I have added comment as well.